src/baitapdateclass/Main.java

```diff
@@ -0,0 +1,68 @@
+package baitapdateclass;
+
+import java.util.Scanner;
+
+public class Main {
+
+	public static void main(String[] args) {
+		int date, month, year;
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Nhập ngày: ");
+		date = scanner.nextInt();
+		System.out.println("Nhập tháng: ");
+		month = scanner.nextInt();
+		System.out.println("Nhập năm: ");
+		year = scanner.nextInt();
+
+		switch (month) {
+		case 1:
+		case 3:
+		case 5:
+		case 7:
+		case 8:
+		case 10:
+		case 12:
+			if (date > 0 && date <= 31) {
+				System.out.println("Ngày tháng năm hợp lệ");
+				break;
+			} else {
+				System.out.println("Ngày tháng năm không hợp lệ");
+				break;
+			}
```

Comment:
```suggestion
			if (date > 0 && date <= 31) {
				System.out.println("Ngày tháng năm hợp lệ");
			} else {
				System.out.println("Ngày tháng năm không hợp lệ");
			}
			break;
```

hoặc nếu em để break trong if (line 27) thì ko cần `else` chỉ cần line 29 là đủ 

```suggestion
			if (date > 0 && date <= 31) {
				System.out.println("Ngày tháng năm hợp lệ");
				break;
			}
			System.out.println("Ngày tháng năm không hợp lệ");
			break;
```